vnext/Shared/WinRTWebSocketResource.cpp

```diff
@@ -67,7 +70,18 @@ auto resume_in_queue(const Mso::DispatchQueue &queue) noexcept {
   };
 
   return awaitable{queue};
+} // resume_in_queue
+
+string HResultToString(hresult_error const &e) {
+  std::stringstream hexCode;
+  hexCode << "[0x" << std::hex << e.code() << "] ";
+  return hexCode.str() + winrt::to_string(e.message());
```

Comment:
>winrt::to_string(e.message()); [](start = 25, length = 30)

(nit) why not place this into stringstream, and then do stringstream.str() only once
Should save on couple of string objects creation/destruction

Reply:
Good catch.

I'll update in master in a subsequent change.